src/test/java/emissary/test/core/junit5/ExtractionTest.java

```diff
@@ -54,6 +54,8 @@ public abstract class ExtractionTest extends UnitTest {
                     KffDataObjectHandler.SET_FILE_TYPE);
     @Nullable
     protected IServiceProviderPlace place = null;
+    @SuppressWarnings("NonFinalStaticField")
+    private static String systemOsRelease = "";
```

Comment:
Will this cause a thread safety issue when tests run in parallel?

Reply:
Since this is a value based on the machine running the suite, once set I don't believe this value would change during the entirety of the test run. It would be either the value or `null` and even in a race condition it would race to the same value.

It seemed like a `BeforeAll` to me, but then there would be multiple switch statements needed while we await a schema with a set of valid values.

Reply:
My thought was what Jason said. It would be null or the value, since it's based on the environment, and that would be the same, and then in a race condition, go to that same value.

Using a `BeforeAll` would require multiple switch statements, due to the existing one checking the answer file attribute as well as the OS environment at once the first check. We could split out the check to be `BeforeAll` if we wanted to be safe, would just add the OS env check before tests.

Reply:
due to the fact that we're _probably_ going to add other checks for OS info, I figured I'd move it to a `BeforeAll` for ease in the future.